tetris.py

```diff
@@ -60,6 +61,7 @@ def run_gui():
     topbar = tkinter.Frame(root, bg=D_GREY, relief="ridge")
     topbar.grid(row=0, columnspan=2, sticky="we")
 
+    global topbar_time
```

Comment:
Are these needed? Figure out which ones aren't, and delete them.

Reply:
Hmm, from what I can see, all of them are needed.